prombench/manifests/prombench/benchmark/6_loadgen.yaml

```diff
@@ -48,6 +48,33 @@ data:
         - expr: histogram_quantile(0.99, sum by(path, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(path, method, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(instance, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
+      - name: arithmetic_operation
+        interval: 10s
+        type: instant
+        queries:
+        - expr: rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m]) + 100  
+        - expr: rate(go_memstats_frees_total[5m]) * 60
+        - expr: rate(codelab_api_requests_total{method=~"GET|POST"}[5m]) - rate(codelab_api_request_duration_seconds_sum{method=~"GET|POST"}[5m])
+        - expr: rate(go_gc_duration_seconds_sum{job=~"fake-webservers-[0-9]+"}[5m]) / rate(go_gc_duration_seconds_count{job=~"fake-webservers-[0-9]+"}[5m])        
+        - expr: sum by (instance, job) (rate(codelab_api_request_errors_total[5m])) / sum by (instance, job) (rate(go_memstats_mallocs_total[10m]))
+      - name: logic_operator
+        interval: 10s
+        type: instant
+        queries:
+        - expr: codelab_api_request_duration_seconds_bucket{method="GET"} or codelab_api_request_duration_seconds_bucket{method="POST"}
+        - expr: codelab_api_request_duration_seconds_sum{status="200"} or codelab_api_request_duration_seconds_sum{status="500"}
+        - expr: codelab_api_request_duration_seconds_bucket{status="200"} and codelab_api_request_duration_seconds_bucket{method="GET"}
+        - expr: codelab_api_request_duration_seconds_count{method="POST"} and codelab_api_request_duration_seconds_count{status="500"}
```

Comment:
I don't see much point in doing multiple expressions that are essentially the same.
`or` is different to `and`, but after that you could have an `/`, taking the ratio of errors to all requests for instance.

Reply:
I realized a couple of things since this comment: you have `/` in "arithmetic operation" above, but this `and` is never going to return anything because the labels on each side are different. We want the benchmark queries to make sense.